dpbench/benchmarks/npbench/mandelbrot1/mandelbrot1_numba_dpex_n.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: 2017 Nicolas P. Rougier
+# SPDX-FileCopyrightText: 2021 ETH Zurich and the NPBench authors
+# SPDX-FileCopyrightText: 2022 - 2023 Intel Corporation
+#
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Looks like this file was missed when replacing nb.njit with dpjit.

Reply:
Thank you!

Reply:
It is still Failed Execution, btw